pkg/services/rpcsrv/server_test.go

```diff
@@ -2617,20 +2891,59 @@ func encodeBlock(t *testing.T, b *block.Block) string {
 	return base64.StdEncoding.EncodeToString(w.Bytes())
 }
 
+func encodeTx(t *testing.T, tx *transaction.Transaction) string {
+	w := io.NewBufBinWriter()
+	tx.EncodeBinary(w.BinWriter)
+	require.NoError(t, w.Err)
+	return base64.StdEncoding.EncodeToString(w.Bytes())
+}
+
+func newTxWithParams(t *testing.T, chain *core.Blockchain, code opcode.Opcode, validUntilIncr uint32, systemFee int64,
+	networkFeeMultiplier float32, addAttrNotValidBeforeT bool) *transaction.Transaction {
+	priv0 := testchain.PrivateKeyByID(0)
+	acc0 := wallet.NewAccountFromPrivateKey(priv0)
+
+	addNetworkFee := func(tx *transaction.Transaction) {
+		size := io.GetVarSize(tx)
+		netFee, sizeDelta := fee.Calculate(chain.GetBaseExecFee(), acc0.Contract.Script)
+		tx.NetworkFee += netFee
+		size += sizeDelta
+		tx.NetworkFee += int64(size) * chain.FeePerByte()
+		tx.NetworkFee = int64(float32(tx.NetworkFee) * networkFeeMultiplier)
+	}
+
+	height := chain.BlockHeight()
+	tx := transaction.New([]byte{byte(code)}, 0)
+	tx.Nonce = height + 1
+	tx.ValidUntilBlock = height + validUntilIncr
+	tx.Signers = []transaction.Signer{{Account: acc0.PrivateKey().GetScriptHash()}}
+	tx.SystemFee = systemFee
+	if networkFeeMultiplier > 0 {
```

Comment:
Remove this check. If multiplier is 0, then the resulting fee is 0.

Reply:
done